Add Article to New York Times Clone

francisuloko opened the pull request:
I have added the header section
The feature image section
I used flex box to position text over inflation image

francisuloko committed:
Add index.html to new your times clone

francisuloko committed:
Add stylesheet to new york times clone

francisuloko committed:
Add README file to new york times clone

francisuloko committed:
Add linters to new york times clone

francisuloko committed:
Add header with nav elements

francisuloko committed:
Add styles for header and nav elemets

francisuloko committed:
Fix linter new line and indentation error messages

francisuloko committed:
Fix linter new line error messages

francisuloko committed:
Add article contents: text, image, icons

francisuloko committed:
Add images folder and article resources

francisuloko committed:
Add hidden files containing site resources

francisuloko committed:
Add headline and feature image

francisuloko committed:
Add style rules to article, headline and feature image

francisuloko committed:
Add new your times logo

francisuloko committed:
Add resources to gitignore file

francisuloko committed:
Position text on background image in inflation section

francisuloko committed:
Add styles for inflation theory section

francisuloko committed:
Add new your times logo snip

francisuloko committed:
Clean up index.html and style.css

francisuloko committed:
Hide article resources

francisuloko committed:
Clean stylesheet

OmarRamoun reviewed (commented):
Linter should be set up correctly for this repository

OmarRamoun commented on README.md:
```diff
@@ -0,0 +1,51 @@
+
+## Built With
+
+- Languages - HTML, CSS
+- Frameworks - boostraps
+- Technology - Linters
```
add the technologies used for this properly: Frameworks: N/A, Technology: linters, git, github

francisuloko replied:
Updated. Thanks.

OmarRamoun commented on README.md:
```diff
@@ -0,0 +1,51 @@
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
add a link to your partner's profile: https://github.com/mrramoun

francisuloko replied:
Updated. Thanks.

OmarRamoun reviewed (commented):
The repo needs some edits :recycle:

francisuloko committed:
Fix linters stylelint error

francisuloko committed:
Rename invalid image title

francisuloko committed:
Add headins to article and section

francisuloko committed:
Fix linters unexpected unit error

francisuloko committed:
Fix linters heading error

francisuloko committed:
Clean stylesheet codes

francisuloko committed:
Fix linters add header to section error

francisuloko committed:
Set iframe border to 0 with CSS

francisuloko committed:
Update README.md

Applying Code Review Changes

Updated "built with" section and added linter, git and github to list of technologies.
I mentioned my coding partner in Ramoun in Acknowledgement

francisuloko committed:
Add article footer/footnote

francisuloko committed:
Add article footer styles

francisuloko committed:
Add the new york times logo font

francisuloko committed:
Merge branch 'add-article' of https://github.com/francisuloko/new-york-times-clone into add-article
Worked on code review recommendations so remote repo is ahead of local
repo and can be safely merged

francisuloko committed:
Add all site images to article

francisuloko committed:
Add editor section to article

francisuloko committed:
Add editor section styles

francisuloko committed:
Fix linters stray div and no header in section error

francisuloko committed:
Fix linters newline error and duplicate style rule

francisuloko committed:
Fix linters unexpected extra colon error

francisuloko committed:
Add site footer

francisuloko committed:
Add site footer styles

francisuloko committed:
Fix linters h3 and hr child of ul error message

francisuloko committed:
Fix linters end of line error

francisuloko committed:
Delete links.txt

File already in .gitignore file

francisuloko committed:
Delete article.txt

File already in .gitignore file

francisuloko committed:
Update README.md

Update list of technology used to work on this project

francisuloko committed:
Update README.md

Updated acknowledgement section to include stand up team members

SmithVyne reviewed (changes requested):
Changes Required 👍 👍 
Hello team, you have worked really hard on this project, but I can see that you haven't yet completed it. 
[REQUIRED] - Images are missing in the section `The Theory of Inflation`. Try to add them.
[REQUIRED] - Also, your git-flow is wrong. You need a `development` also. Try to fix that.
Complete these and other problems then submit them for a review. 🎖️

francisuloko committed:
Add image to the theory of inflation section

francisuloko committed:
Style image in the theory of inflation section

francisuloko committed:
Fix linters error to add line after comment

francisuloko commented:
Hi Smith, thank you for your recommendation. I have applied the required changes and submitted my code for a review. Thank you.

christianotieno commented on index.html:
```diff
@@ -0,0 +1,474 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
+    <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"> -->
```
## Remove commented out code
- [ ] I'd suggested It’s better to delete these **commented pieces of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
```suggestion
```

christianotieno commented on index.html:
```diff
@@ -0,0 +1,474 @@
+                    <span class="site-name">space & cosmos</span>
+                </div>
+                <div>
+                    <!-- <img src="images/thenewyourtimes.png" alt="The New York Times" width="213" height="31"> -->
```
## Remove commented out code
- [x] I'd suggested It’s better to delete these **commented pieces of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
```suggestion
```

christianotieno commented on index.html:
```diff
@@ -0,0 +1,474 @@
+                <h2>Advert Space</h2>
+            </section>
+            <section class="text headline">
+                <h2>Space Ripples Reveal Big Bang’s Smoking Gun</h2>
```
## Improve the styling on the `Space Ripples Reveal Big Bang’s Smoking Gun` section
- [x] Kindly remove the spacing in between the main image of the `Space Ripples Reveal Big Bang’s Smoking Gun` article and the heading itself as it is not supposed to be there.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/francisuloko/new-york-times-clone/add-article/index.html)|
|:--:|:--:|
|![main-image](https://user-images.githubusercontent.com/22345961/108378250-e2ce9080-7215-11eb-8e20-ca3a5c8963bf.png)|![main-image-clone](https://user-images.githubusercontent.com/22345961/108378274-e6faae00-7215-11eb-87a4-2cf820874822.png)|

christianotieno commented on index.html:
```diff
@@ -0,0 +1,474 @@
+                <p>Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.</p>
+            </section>
+            <section class="feature">
+                <h3>The Theory of Inflation</h3>
```
## Improve the `Theory of Inflation` Section
- [x] Images for this section are missing as evident in the contrasting images below. Kindly add them. 

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/francisuloko/new-york-times-clone/add-article/index.html)|
|:--:|:--:|
|![theory-of-inflation](https://user-images.githubusercontent.com/22345961/108377052-ac444600-7214-11eb-86b0-5bdf8bb213f5.png)|![theory-of-inflation-clone](https://user-images.githubusercontent.com/22345961/108377068-b23a2700-7214-11eb-8eb4-34677655d76c.png)|

francisuloko replied:
Thank you for your feedback Christian, your guide was really helpful. I have applied the recommended changes. Although on my local machine the image displays correctly,  but  I have since updated the file path this time.

christianotieno reviewed (changes requested):
## Changes Required ♻️ 
Hello @francisuloko 👋 

Great job with the changes. The layout is turning out just fine. This review change suggestion touches on the previously issued changes on the `theory of inflation` section. More details about this down below 👇 
### Happy coding 💻 
#### Christian Otieno (CR)

francisuloko committed:
Fix inflaction section image size

francisuloko committed:
Apply code review recommendations

francisuloko committed:
Style code review recommendations

francisuloko committed:
Update README.md

Add live demo link

SpaYco reviewed (approved):
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

francisuloko commented:
Thank you Spayco for approving this code, it feels great. I will remember to merge the code. Have fun coding too.
